CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
 # Change Log
 
+
+## v0.4.9
```

Comment:
Just use `## [Unreleased]` as version number for the time being. I'll change that to the actual version number upon release.

fyi. I usually create a new release every time a new feature is added. It will only take a few minutes once this PR is merged.